compiler/src/dotty/tools/dotc/sbt/ExtractDependencies.scala

```diff
@@ -424,13 +424,30 @@ class DependencyRecorder {
       classDependencies.foreach(recordClassDependency(cb, _))
     clear()
 
+  private val _siblingClassfiles = new mutable.HashMap[PlainFile, Path]
```

Comment:
because this is only used from sendToZinc which calls clear() at the end, it could also be a local val inside sendToZinc (after inlining recordClassDependency inside sendToZinc)

Reply:
done